crates/sui/src/client_commands.rs

```diff
@@ -46,6 +46,7 @@ use sui_sdk::sui_client_config::{SuiClientConfig, SuiEnv};
 use sui_sdk::wallet_context::WalletContext;
 use sui_sdk::SuiClient;
 use sui_types::{
+    // balance::BALANCE_CREATE_REWARDS_FUNCTION_NAME,
```

Comment:
It would be better to just remove lines like this from a PR, but if you're working in a stack and this is going to go away soon, it's okay.

Reply:
Uhm, don't know how that snuck in....